### ISSUE TITLE
Display code snippets menu in REPL toolbar

### ISSUE THREAD

**Issue by zooba** [Microsoft/PTVS#2268]
We should have a button in the REPL toolbar showing a certain category of code snippets. This will:

1. Make it obvious that code snippets can be used
2. Provide a way for us to simplify visualization sooner than #748

(Depends on #2264)

**Comment by huguesv** [comment]
We're closing old issues with no recent activity. If this is still important, please open a new issue.
